### ISSUE TITLE
[testing-toolkit-test-cases] provisioning collection fails out of the box with cgscurrency

### ISSUE THREAD

**Issue by lewisdaly** [mojaloop/project#2619]
**Summary**:
The ttk provisioning collection (`testing-toolkit-test-cases/collections/hub/provisioning/`) collection fails out of the box on a new deployment. 

It looks like the test case "Add initial position and limits with different currency - payerfsp CGS " requires a variable to be set in the environment file: `cgscurrency`, which doesn't exist in `https://github.com/mojaloop/testing-toolkit-test-cases/blob/master/environments/hub.json`

I know that that variable exists in the other environment file, so my concern is more: _"what are the sensible defaults here?"_ For the default provisioning collection, the default `hub.json` environment file should lead to a successful run of the collection.

I'm running a mojaloop deployment locally on k3s, and using a local ttk cli to talk to the backend like so:

```bash
npm run cli -- \
	-u http://testing-toolkit.local \
	-i ../testing-toolkit-test-cases/collections/hub/provisioning/ \
	-e ../test-runner/ttk_environment.json \
	--report-format html
```

**Severity**:
Medium

**Priority**:
Medium

**Expected Behavior**
The default provisioning collection should work out of the box with the default values file.

**Steps to Reproduce**
  1. Install a mojaloop switch locally (v13.0.2
  2. Run the provisioning collection with the above command
  3. Observe the failure:
![ttk_failure.png](https://images.zenhubusercontent.com/5cd963af3613073e20e46e60/269293ef-7b03-462f-ab70-2b23ffc1099d)

**Specifications**
  - Component (if known):
  - Version:
  - Platform:
  - Subsystem:
  - Type of testing:
  - Bug found/raised by:

**Notes**:

- Severity when opened:
- Priority when opened:

**Comment by lewisdaly** [comment]
I can't upload the full report html file, as it's too big for github.

**Comment by lewisdaly** [comment]
Other values are also missing: `SIM3_NAME`, `SIM4_NAME`

**Comment by lewisdaly** [comment]
Also `NET_DEBIT_CAP`

**Comment by elnyry-sam-k** [comment]
Thanks @lewisdaly , will discuss this tomorrow..

**Comment by vijayg10** [comment]
Hi @lewisdaly, are you using a matching version of test cases as the mojaloop version?
I mean if you are using mojaloop helm version 13.0.2 then please use the collection files in v13.0.2 tag of the testing-toolkit-test-cases repo.

**Comment by lewisdaly** [comment]
Thanks @vijayg10 I see your point. I was using the latest test cases in `master` - but I would still expect the test cases and environment files to be consistent within the same repo

**Comment by vijayg10** [comment]
Agreed @lewisdaly, the sample environment files should be updated. I will find out the delta and update those files.
